src/depthimage_to_laserscan_slice_node.cpp

```diff
@@ -31,9 +30,18 @@ DepthImageToLaserScanSliceNode::DepthImageToLaserScanSliceNode(ros::NodeHandle&
   depth_to_scan_ = std::make_unique<DepthimageToLaserscanSlice>(
       width, height, fx, fy, cx, cy, 1000.0);
 
-  depth_to_scan_->set_height(0.05, 0.6);
+  double height_min, height_max;
```

Comment:
[declaration](https://github.com/groove-x/depthimage_to_laserscan_slice/blob/master/include/depthimage_to_laserscan_slice/depthimage_to_laserscan_slice_node.hpp#L25-L26) of `range_min_` `range_max_` can also be moved into this .cpp?

Reply:
I also thought that and tried to do so, but couldn't.
They are used [here](https://github.com/groove-x/depthimage_to_laserscan_slice/blob/88c9f31873beb37c7ec9f938100bc377e2db63ea/src/depthimage_to_laserscan_slice_node.cpp#L92-L93).